src/server/server.ts

```diff
@@ -577,6 +577,11 @@ namespace ts.server {
         }
     }
 
+    const localeStr = findArgument("--locale");
+    if (localeStr && validateLocaleAndSetLanguage(localeStr, sys)) {
+        validateLocaleAndSetLanguage(localeStr, sys);

```

Comment:
not sure that I get why do we need to call `validateLocaleAndSetLanguage ` twice